nautobot/users/models.py

```diff
@@ -245,6 +245,7 @@ class ObjectPermission(BaseModel):
                     "users",
                 ]
             )
+            | Q(app_label="admin", model__in=["logentry"])
             | Q(app_label="auth", model__in=["group", "user"])
             | Q(app_label="users", model__in=["objectpermission", "token"])
```

Comment:
I noticed the other day that this is actually wrong - we don't use `auth.user`, we use `users.user`; the most obvious side effect I noticed is that you can't actually define objectpermissions around `users.user` at present. Might make sense to fix that as part of this PR?